Feature/speaker ashley

agorilla opened the pull request:
Ashley speaker implementation.

Tried to remove the background of her pic but it looked not good.
Feel free to try but I'd say it's not needed.

agorilla committed:
Added Ashley to speakers

- photo from Twitter in black and white
- bio from Twitter
- ImageOptim JPGs
- fixed to small photo copyright space in mobile

agorilla committed:
Added fx-fade effect to all speaker list items

anselmh commented:
TBH I don’t like the bio text. It’s hard to read and IMO inappropriate for a conference text. Can we use something from here? http://ashleygwilliams.github.io/about.html

anselmh commented:
Image background is fine btw.

agorilla committed:
Added more detailed bio for Ashley

I used her bio text from JSConf2015 (http://2015.jsconf.us/speakers.html#williams).
If this is a rights problem, we need to write another text but I', not able to do this today.
I'd say lets ask Ashley if we can use the text.

agorilla commented:
@asciidisco Can you ask Ashley if we can use the text from her JSConf2015 speaker section (http://2015.jsconf.us/speakers.html#williams) or if she wants to provide us another one?

Thanks
